src/commands/add.ts

```diff
@@ -89,8 +92,8 @@ export const usage: Usage = () => {
     Add a new route 'howdy'
     {cyanBright ggt add route GET howdy}
 
-    Add a new 'boolean' type field 'published' to an existing model
-    {cyanBright ggt add field post/published:boolean}
+    Clone the \`development\` environment into a new \`staging\` environment
+    {cyanBright ggt add environment staging --clone development}
```

Comment:
I don't see the `--clone` flag implemented anywhere. It looks like we always use the current environment.

Should we just 🔥 the flag?